src/router/authLoader.ts

```diff
@@ -1,21 +1,33 @@
+import 'core-js/stable/atob'
+import { userData } from '@/types'
+import axiosInstance from '@/axiosConfig'
 /**
  * Auth state loader for react-router data routes.
  * @module router/authLoader
  * @see {@link dashboard/Routes}
  */
 
+const emptyUser: userData = {
+  userid: '',
+  email: '',
+  fullname: '',
+  role: '',
+  assignedfismasystems: [],
+}
```

Comment:
do you really need this? it exposes the shape of api data to the public do I would rather avoid it